video_xblock/static/html/wistiavideo.html

```diff
@@ -0,0 +1,10 @@
+<video
+  id="{{ video_player_id }}"
+  muted="muted"
+  src=""
+  class="video-js vjs-default-skin"
+  controls
+  width="829" height="511"
+  data-setup='{ "controlBar": {"volumeMenuButton": {"inline": false, "vertical": true}},  "techOrder": ["wistia"], "sources": [{ "type": "video/wistia", "src": "{{ url }}?controlsVisibleOnLoad=false" }], "playbackRates": [0.5, 1, 1.5, 2] }'
```

Comment:
does it make sense to move `playbackRates` to the context? It is the same for all players.